position/sync/position.js

```diff
@@ -1,9 +1,7 @@
 // var { link } = require('ssb-msg-schemas/util')
 
-function Position ({ positionDetails, reason, body, channel, recps, mentions }) {
-  var content = { type: 'position', positionDetails, reason }
```

Comment:
positions should have reasons, not body?

Reply:
yep